apps/src/codegen.js

```diff
@@ -92,8 +93,11 @@ export function evalWithEvents(apis, events, evalCode = '') {
   // to call, and any arguments.
   const eventLoop = ';while(true){var event=wait();setReturnValue(this[event.name].apply(null,event.args));}';
 
-  interpreter = new Interpreter(
+  // TODO (pcardune): remove circular dependency
```

Comment:
I need to move `codegen` to `lib/tools/jsinterpreter`. At that point, I will probably move a number of these functions onto CustomMarshalingInterpreter and kill off the circular deps.